app/src/main/java/com/simply/birthdayapp/presentation/ui/components/RoundAsyncImage.kt

```diff
@@ -0,0 +1,53 @@
+package com.simply.birthdayapp.presentation.ui.components
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.painter.Painter
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import coil.size.Size
+
+@Composable
+fun RoundAsyncImage(
+    modifier: Modifier = Modifier,
+    data: Any? = null,
+    borderColor: Color,
+    borderWidth: Dp,
+    contentDescription: String? = null,
+    contentScale: ContentScale = ContentScale.Crop,
+    placeholder: Painter? = null,
+    error: Painter? = null,
+) {
+    AsyncImage(
+        model = ImageRequest.Builder(LocalContext.current)
+            .data(data)
+            .size(Size.ORIGINAL)
```

Comment:
Isn't the size ORIGINAL by default?

Reply:
[Removed](https://github.com/SimplyTechnologies/android-internship/pull/14/commits/6e6bbbe912d973edb847dbf38bd7c2cad6601267)

Default seems to be `Size(displayMaxDimension, displayMaxDimension)`.